src/schema/annotation.yaml

```diff
@@ -81,56 +82,14 @@ classes:
     is_a: FunctionalAnnotationTerm
     description: >-
       A pathway is a sequence of steps/reactions carried out by an organism or community of organisms
-    slots:
-      - has_part
-    slot_usage:
-      has_part:
-        range: Reaction
-        required: true
-        description: >-
-          A pathway can be broken down to a series of reaction step
     id_prefixes:
       - KEGG_PATHWAY
       - COG
     exact_mappings:
       - biolink:Pathway
-
-  Reaction:
-    class_uri: "nmdc:Reaction"
-    is_a: FunctionalAnnotationTerm
-    description: >-
-      An individual biochemical transformation carried out by a functional unit of an organism, 
-      in which a collection of substrates are transformed into a collection of products.
-      Can also represent transporters.
-    slots:
-      - direction
-      - is_balanced
-      - is_diastereoselective
-      - is_fully_characterized
-      - is_stereo
-      - is_transport
-      - left_participants
-      - right_participants
-      - smarts_string
-    id_prefixes:
-      - KEGG.REACTION
-      - RHEA
-      - MetaCyc
-      - EC
-      - GO
-      - MetaNetX
-      - SEED
-      - RetroRules
-    exact_mappings:
-      - biolink:MolecularActivity
-
-  ReactionParticipant:
-    class_uri: "nmdc:ReactionParticipant"
-    description: Instances of this link a reaction to a chemical entity participant
-    slots:
-      - chemical
-      - stoichiometry
-      - type
+    todos:
+      - If we reverted to including Reaction in the schema, then a Reaction would be a reasonable part_of a Pathway
+      - is Pathway instantiated in an MongoDB collection? Aren't Pathways searchable in the Data Portal?
```

Comment:
I don't think this needs to prevent merging this PR. Can you make an issue for this @turbomam 
@aclum should check.  But I think it's a different collection that manages that pathway?